staff/gextremera/leadtech-site-semantic-markup/about-us.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About Us</title>
+</head>
+<body>    
+    <nav>
+        <ul>
+            <li><a href="https://leadtech.com/">Leadtech</a></li>
+            <li><a href="https://leadtech.com/about-us">About Us</a></li>
+            <li>
+                <a href="https://leadtech.com/work-with-us">work with us</a>
+                <ul>
+                    <li><a href="https://leadtech.com/work-with-us/backend">BackEnd</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/bi">BI</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/content-and-copywriting">Content and copywriting</a></li>
+                    <li><a href="https://leadtech.com/work-with-us/customer-service"></a>CS</li>
+                </ul>
+            </li>
+            <li><a href="https://leadtech.com/entrepreneur">Entrepreneurs</a></li>
+            <li><a href="https://leadtech.com/contact">Contact</a></li>
+        </ul>
+    </nav>
+    <header>
+        <h1>About Us</h1>
+        <p>A pioneer in online project management, leadtech was born in 2007 to foster innovation and creativity across a multitude of industries.</p>
+    </header>
+    <section>
```

Comment:
Most of layouts contains `header`, `main` & `footer` as main containers. The rest of tags are inside them